app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :posts_id
```

Comment:
- [ ] Please make sure Comment model has a method that updates the comments counter for a post.

```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```